R/MolgenisDriver.R

```diff
@@ -110,8 +115,13 @@ setMethod(
 #' package (`driver.version`) and the version of the underlying client
 #' library (`client.version`).
 #'
-#' @import methods
 #' @export
-setMethod("dsGetInfo", "MolgenisDriver", function(dsObj, ...) { # nolint
-  # TODO implement
-})
+methods::setMethod(
+  "dsGetInfo", "MolgenisDriver",
+  function(dsObj, ...) { # nolint
+    return(list(
+      driver.version = packageVersion("DSI"),
+      client.version = packageVersion("DSMolgenis")
```

Comment:
Shouldn't these two be the other way around?

Reply:
owh.. unsure! looking it up